app/javascript/src/Policies/styles/policies.scss

```diff
@@ -11,13 +11,16 @@ $subtleColor: #ddd;
 
 .PoliciesWidget {
   position: relative;
-  width: 510px;
-  margin: 0 auto;
   line-height: 1.5;
   padding-bottom: 2rem;
 
+  .PolicyChain, .PolicyRegistryList, .PolicyConfiguration {
+    width: 510px;
```

Comment:
pxs are forbidden where i come from

Reply:
I've just moved it from here: https://github.com/3scale/porta/pull/602/files/bb11cd4c3455502468b1011e9c3c4a784123f9f4..331a9c841cc5ed34614ec7cd9c68c2d4f310e148#diff-dc7eadc737110a7b487300a9ec5b4f00L14

But better if you or @damianpm polish the styling afterwards